core/templates/pages/library-page/library-footer/library-footer.component.ts

```diff
@@ -19,22 +19,24 @@
 import { Component } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
 import { WindowRef } from 'services/contextual/window-ref.service';
-import { LibraryPageConstants } from '../library-page.constants';
+import { LibraryPageConstants, libraryPathToModes } from '../library-page.constants';
+
+type libraryPathToModesKeys = keyof typeof libraryPathToModes;
```

Comment:
```suggestion
type libraryPathToModesKeys = (
  keyof typeof LibraryPageConstants.LIBRARY_PATHS_TO_MODES);
```
Using this won't require you to change constants file

Reply:
Done